pkgs/tools/X11/xssstate/default.nix

```diff
@@ -4,29 +4,34 @@
 , libX11
 , libXScrnSaver
 }:
-stdenv.mkDerivation rec {
+let
+  version = "1.1-unstable-2022-09-24";
+in
+stdenv.mkDerivation {
   pname = "xssstate";
```

Comment:
I think we should convert this to finalAttrs instead.

Reply:
Converted to `finalAttrs`.